File: Weekend Exercises/Python Version/task_five.py
word = input("Enter a word: ")

count = 1

for letter in word.lower():

	if (letter == "a" or letter == "e" or letter == "i" or letter == "o" or letter == "u"):

		break;

	count += 1;

print("The position of first vowel is", count)

File: Weekend Exercises/Java Version/TaskFive.java
import java.util.Scanner;
public class TaskFive{

	public static void main (String [] args){

		Scanner input = new Scanner (System.in);

		System.out.println("Enter a word: ");
		String words = input.nextLine();
		String word = words.toLowerCase();

		int index = 0;

		for (; index < word.length(); index++){

			char letter = word.charAt(index);
			if (letter == 'a' || letter == 'e' || letter == 'i' || letter == 'o' || letter == 'u'){
				break;
			}
		}

		System.out.printf("The position of first vowel in %s is %d%n", word, index + 1);
	}
}